chain-abci/tests/abci_app.rs

```diff
@@ -788,9 +788,10 @@ fn get_account(account_address: &RedeemAddress, app: &ChainNodeApp<MockClient>)
     let account_key = to_stake_key(&StakedStateAddress::from(*account_address));
     let state = app.last_state.clone().expect("app state");
     println!("committed root hash: {:?}", &state.last_account_root_hash);
-    let items = app
-        .accounts
-        .get(&state.last_account_root_hash, &mut [account_key.clone()]);
+    let items = app.accounts.get(
+        &app.uncommitted_account_root_hash,
+        &mut [account_key.clone()],
+    );
```

Comment:
there should be now `get_one` method -- most tests were written for the earlier before this existed (and weren't updated), so for new code if it only looks up one value, it's preferable instead of wrapping in this `&mut [..]` and unwrapping the result from a map